src/utils/logger.ts

```diff
@@ -4,17 +4,17 @@ import { resolve } from 'path';
 import { RemoteConsole } from 'vscode-languageserver/node';
 import { fsProvider } from '../fs-provider';
 
-function getEnv() {
-  if (typeof process !== undefined) {
-    return process.env;
-  } else {
-    return {};
-  }
-}
+// function getEnv() {
```

Comment:
we need to uncomment this code once we ready to go

Reply:
done!